_topic_map.yml

```diff
@@ -270,7 +270,7 @@ Topics:
 ---
 Name: Web console
 Dir: web_console
-Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated,openshift-online
+Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated,openshift-online,openshift-aro
 Topics:
 - Name: Accessing the web console
   File: web-console
```

Comment:
Many of the topics here which are marked `openshift-enterprise,openshift-webscale,openshift-origin` should also be marked `openshift-aro`.  That may apply in other areas of the file too.

Reply:
This needs to be gone through carefully - certainly topics in "Understanding authentication" also suffer from this.